scripts/download-vsix-from-circleci.js

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
+
+const shell = require('shelljs');
+shell.set('-e');
+shell.set('+v');
+
+const { getVsixName, getLocalPathForDownload } = require('./publish-utils');
+const CIRCLECI_API_URI = 'https://circleci.com/api/v1.1';
+const circleciToken = process.env['CIRCLECI_TOKEN'];
+const circleciBuild = process.env['CIRCLECI_BUILD'];
+const nextVersion = process.env['SALESFORCEDX_VSCODE_VERSION'];
```

Comment:
Is there a reason we use environment variables for these values? They change on a weekly basis (or even more if we need to hotfix). If we were to eventually utilize this script in an automated environment we'd need to change these in some settings portal every time.

Reply:
That is a good point. Why do we need SALESFORCEDX_VSCODE_VERSION when circleci-to-vscode doesn't?

Reply:
It's true that this script is not optimized for running on an automation environment (e.g. GH Actions) but I think those updates should be done once we start working on that automation piece. Doing it right now would be to optimize for something we don't exactly know how we'll structure.

The reason we use `SALESFORCEDX_VSCODE_VERSION ` here and not on the circleci-to-vscode utility is because we want to validate this is executed in the correct branch since the publish process makes updates the files (e.g. sha generation) that need to be checked in and merged.

Reply:
Just an idea: We could turn this into a task that accepts the vscode version as an input. The default value could be the latest vscode version. This would still keep the check, but easily allow for an override.